civiform/civiform
# [Program Migration] Set up basic export UI

### Description

Part of epic #7087 to enable admins to export and import programs between instances. This PR sets up an "Export" tab on the admin side. That tab displays a form with all the currently active programs, lets the admin select one and then click "Download". Right now, that "Download" button doesn't do anything. A future PR will make that Download button download a JSON that represents the program and all its questions.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


#### New Features


### Instructions for manual testing

- Disable the `PROGRAM_MIGRATIONS` flag. Verify the "Export" tab doesn't appear in the admin UI.
- Enable the `PROGRAM_MIGRATIONS` flag. Verify the "Export" tab appears in the admin UI. Click it and verify you're presented with the list of active programs. Verify you can select one and click "Download".

### Issue(s) this completes

Part of epic #7087
